sdk/netapp/Azure.ResourceManager.NetApp/src/autorest.md

```diff
@@ -85,14 +85,12 @@ prepend-rp-prefix:
   - ReplicationObject
   - ReplicationSchedule
   - VolumeStorageToNetworkProximity
+  - VolumeProperties
```

Comment:
This looks like did not have any effect on the SDK, could we remove this?

Reply:
fixed